examples/example_gather_metrics.py

```diff
@@ -0,0 +1,61 @@
+import asyncio
```

Comment:
this probably doesn't need its own example anymore- we could just set the `metrics_output_path` in one of the other examples with a comment about what it does